samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+     - | Set IP address
+       | <my ip> <prefix len>
+       | Example - setip 2001:db8::2 64
+       | Example - setip 192.0.2.2
+   * - tcp
+     - | Upload/Download TCP data
+   * - udp
+     - | Upload/Download UDP data
+   * - version
+     - | **zperf** version
+
+The following is an example of the ``zperf`` command:
+
+.. code-block:: console
+
+   zperf <udp/tcp> <upload/download> <dest ip> [<dest port> <duration> <packet size>[K] <baud rate>[K|M]]
+
+Testing
+=======
+
+|test_sample|
+
+1. |connect_kit|
+#. |connect_terminal|
+#. Scan for the Wi-Fi networks in range using the following command:
+
+   .. code-block:: console
+
+      wifi scan
+
+   The output should be similar to the following:
+
+   .. code-block:: console
+
+      Scan requested
+
+      Num  | SSID                             (len) | Chan (Band)   | RSSI | Security        | BSSID             | MFP
+      1    | abcd                             8     | 11   (2.4GHz) | -18  | WPA2-PSK        | xx:xx:xx:xx:xx:xx | Disable
+      2    | efgh                             7     | 40   (5GHz  ) | -21  | OPEN            | xx:xx:xx:xx:xx:xx | Disable
+      3    | mnopq                            8     | 6    (2.4GHz) | -29  | OPEN            | xx:xx:xx:xx:xx:xx | Disable
+      6    | stuvwx                           24    | 11   (2.4GHz) | -47  | WPA3-SAE        | xx:xx:xx:xx:xx:xx | Required
+
+#. Connect to your preferred network using the following command:
+
+   .. code-block:: console
+
+      wifi connect <SSID> <passphrase>
+
+   ``<SSID>`` is the SSID of the network you want to connect to, and ``<passphrase>`` is its passphrase.
+
+#. Check the connection status after a while, using the following command:
+
+   .. code-block:: console
+
+      wifi status
+
+   If the connection is established, you should see an output similar to the following:
+
+   .. code-block:: console
+
+      Status: successful
+      ==================
+      State: COMPLETED
+      Interface Mode: STATION
+      Link Mode: WIFI 4 (802.11n/HT)
+      SSID: xxxxx
+      BSSID: xx:xx:xx:xx:xx:xx
+      Band: 2.4GHz
+      Channel: 11
+      Security: WPA2-PSK
+      MFP: Optional
+      RSSI: -28
+      Beacon Interval: 100
+      DTIM: 1
+      TWT: Not supported
+
+#. Initiate a ping and verify data connectivity using the following commands:
+
+   .. code-block:: console
+
+      net ping <resolved hostname>
+
+   See the following example:
+
+   .. code-block:: console
+
+      net ping 192.168.1.1
+      PING 192.168.1.1
+      28 bytes from 192.168.1.1 to 192.168.1.1: icmp_seq=1 ttl=64 time=0 ms
+      28 bytes from 192.168.1.1 to 192.168.1.1: icmp_seq=2 ttl=64 time=0 ms
+      28 bytes from 192.168.1.1 to 192.168.1.1: icmp_seq=3 ttl=64 time=0 ms
+
+#. Run the ``iperf`` commands (TX) on the peer device and the ``zperf`` commands (RX) on the sample as shown in the following table.
+   The Throughput sample test cases require additional software, such as the Wi-Fi **iperf** application.
```

Comment:
The last 2 sentences here should be explained before the test steps. It is essential background information of how the sample is supposed to work. So I would move it earlier under the sample description.

Ideally, you should add a **very** simple block diagram, with the nRF7002, the AP, the peer device, etc. It should show what entity run the zperf tool, what entity runs the iperf, what IP protocols are supported, etc.

That part should also explain the configuration. For example, to test TX, you configure zperf on TX and peer on RX, etc.